db/fixtures/ae_datastore/ManageIQ/Deployment/ContainerProvider/System/StateMachine/Deployment.class/__methods__/add_provider.rb

```diff
@@ -37,13 +32,12 @@ def add_provider
       :auth_key           => token)
 
     $evm.log(:info, "result: #{result}")
-    if result[0]
+    if result
```

Comment:
Would be useful to have a more descriptive variable name than `result`.  Looks like this variable would always contain the new provider.